docs/products/kafka/kafka-connect/concepts/jdbc-source-modes.rst

```diff
@@ -0,0 +1,22 @@
+JDBC source connector modes
+===========================
+
+
+JDBC source connector extracts data from a relational database, such as PostgreSQL or MySQL, and pushes it to Apache Kafka where can be transformed and read by multiple consumers. 
+
+This connector type periodically queries the table(s) to extract the data, and can be configured in four main **modes**:
+
+* ``bulk``: the connector will query the full table every time retrieving all rows and publishing them into the Apache Kafka topic
+* ``incrementing``: the connector will query the table appending a `WHERE` condition based on a **incrementing column**. This requires the presence of a column containing an always growing number (like a series). The incrementing column is used to check which rows have been added since last query. 
```

Comment:
Now as the headers we use what should be put under ``mode`` property when executing commands, but I think it would be better to use proper header and be specific that the ``mode`` should be set to ``bulk`` or other string.

Reply:
Moved to headers and expanded to cover examples